src/jit/compiler.h

```diff
@@ -252,8 +252,9 @@ class LclVarDsc
     unsigned char lvStackByref : 1;  // This is a compiler temporary of TYP_BYREF that is known to point into our local
                                      // stack frame.
 
-    unsigned char lvArgWrite : 1; // variable is a parameter and STARG was used on it
-    unsigned char lvIsTemp : 1;   // Short-lifetime compiler temp
+    unsigned char lvArgWrite : 1;         // there is at least one STLOC or STARG on this local
+    unsigned char lvMultipleArgWrite : 1; // there is more than one STLOC on this local
```

Comment:
Bringing in IL seems like a good improvement, so `lvSingleILStoreOp` and `lvMultipleILStoreOp`? And then similarly for the local bools that get introduced....